verilog/parser/verilog_token_classifications.cc

```diff
@@ -161,6 +161,29 @@ bool IsIdentifierLike(verilog_tokentype token_type) {
     case verilog_tokentype::TK_Sskew:
     case verilog_tokentype::TK_Stimeskew:
     case verilog_tokentype::TK_Swidth:
+
+      // These are from KeywordIdentifier (verilog.y)
```

Comment:
It would probably be justified to re-tag these leaf tokens in verilog.y in the `KeywordIdentifier` production rule, and then never have to worry about replicating this logic.

Reply:
what do you mean with "re-tag" ?

Reply:
Alright, did this now, and the parser test is right to point out that whatever we get out of that is not the same as the original token sequence :)
https://github.com/chipsalliance/verible/runs/5338770908?check_suite_focus=true#step:7:492